ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4429,3 +4429,20 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: programTenantList
+    request_path: "{{ program_service_prefix }}/v1/tenant/list"
+    upstream_url: "{{ program_service_url }}/program/v1/tenant/list"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: 'userUpdate'
```

Comment:
it is having a wrong ACL @harshavardhanc @twsvaishali what is the intent of this API?